### PR TITLE
Crypto stats final settings

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this project we:

- Used coins stats API to get numeric values based 
- retrieved from the API and stored in the Redux store
- are filtering the data that you retrieve from the API using a Filter stateless component.
- had a unique route within the SPA for every page 
- deployed and accessible online using (link to the online version of the app to your README file.)
- Created unit tests for pure functions in Redux code
- Created integration tests for the application using the React Testing Library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +165,7 @@ npx eslint .
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/Bleu-Yves-Sopoude/Bookstore/issues).
+Feel free to check the [issues page](https://github.com/Bleu-Yves-Sopoude/crypto-stats/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (OLIPLICHE) [inline README.md:171]: - Hi, can you move to `acknowledgements` section to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/)?

File: src/redux/tests/Filter.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
-import { filteredCurrencyLists } from '../currencies/currencySlice.js';
+import { filteredCurrencyLists } from '../currencies/currencySlice';
 
 const Filter = () => {
```
Comment (OLIPLICHE) [inline src/redux/tests/Filter.js]: - [ ] Hello.  I found out that you have created a group of files inside of the test folder, and you tested them. These files in the test folder do not link to your app. The work you did is not what the project requirements are saying. 
Here are the requirements for testing:
##### **Testing**

1. App has unit tests for several components of the app and some pure functions in Redux code.
1. App has integration tests using React Testing Library
   * Student can perform a mocking data fetching and Redux Store connection.
 
👉 I will advise you to follow the requirements as it is

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The project contains a `Home` page and a `Detail` page 👍 
- The student recorded a video presenting the features of the project 👍 
- The project is deployed properly 👍 
- The project is a single page application (SPA) using `react-router-dom` 👍 

## Required Changes ♻️
- Please can you remove the commented codes, because it makes your code look dirty. 
Capture


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CrptoDetails.test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
 import configureStore from 'redux-mock-store';
 import CryptoDetails from './CryptoDetails';
-import '@testing-library/jest-dom'
+import '@testing-library/jest-dom';
 
 const mockStore = configureStore([]);
```
Comment (TadesseDev) [inline src/components/CrptoDetails.test.js:9]: - [Optional] Dear @Bleu-Yves-Sopoude, I notice you have moved your test files to the `src/components/` location, this is not the standard way of organizing your tests. Usually, it is recommended to save your test in the `src/test/` folder.

**Review by TadesseDev — APPROVED** [review]
### Hi @Bleu-Yves-Sopoude 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions
- [Optional] Dear @Bleu-Yves-Sopoude, I can see you have a commit message with repeated names (`Fix linter` seven times) and some others are not so clear about what has been achieved in a specific commit. Please try to be more descriptive in your commit messages, this will help you debug your code and understand what happens when. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**